etl/load.py

```diff
@@ -69,6 +108,18 @@ def load_monitoring_location(db_user, db_password, connect_str, mon_loc):
         connect.commit()
```

Comment:
For support, errors from cx_Oracle.connect could be caught and better logged.
Not critical, can be a refactor at some point later...

Reply:
ah, I see now that this expection handled in the calling method...